app/main/controller/audio_transcription_controller.py

```diff
@@ -45,13 +46,19 @@ def safely_handle_transcription_job(callback):
 @api.route('/')
 class AudioTranscriptionResource(Resource):
     def aws_start_transcription(self, jobName, audioUri):
-        return transcribe.start_transcription_job(
-            TranscriptionJobName=jobName,
-            IdentifyLanguage=True,
-            Media={
-                'MediaFileUri': audioUri
-            }
-        )
+        try:
+            return transcribe.start_transcription_job(
+                TranscriptionJobName=jobName,
+                IdentifyLanguage=True,
+                Media={
+                    'MediaFileUri': audioUri
+                }
+            )
+        except (BotoCoreError, ClientError) as e:
+            if 'ConflictException' not in str(e):
+                return transcribe.get_transcription_job(TranscriptionJobName=jobName)
+            else:
+                raise
```

Comment:
Is the goal here try to do the transcription, surface conflict exceptions, but swallow all other BotoCore and client errors?  Seems like there might be other Boto and client errors we would want to surface?

Or, if the goal is, if it is Conflict because job already exists, get the existing job and return it,  it seems like it should be `if 'ConflictExecption' in str(e)` ? (remove the not?)